src/ResultsActivity/elements/PayOptions/PayOptions.js

```diff
@@ -52,12 +56,14 @@ function PayOptions({ offenderDetails, updateOffenderAttendance, date, openModal
   return (
     <Fragment>
       <Option data-qa="pay-option">
-        {offenderNo &&
+        {!isPaying &&
+          offenderNo &&
           eventId && (
             <Radio onChange={payOffender} name={offenderNo} value="pay" checked={selectedOption === 'pay'}>
               <VisuallyHidden>Pay</VisuallyHidden>
             </Radio>
           )}
+        {isPaying && <Spinner title="Paying" height="40px" width="40px" />}
       </Option>
```

Comment:
Should we use govuk spacing here? I think it may be 7 or something...

Reply:
Yeah you're right @Nimphal .  It's supposed to match the size of the radio buttons, which use spacing point 7.  Will update, cheers.